app/src/main/res/layout/activity_main.xml

```diff
@@ -40,7 +40,8 @@
             android:fitsSystemWindows="true"
             app:headerLayout="@layout/nav_header"
             app:itemTextColor="@color/nav_view_colors"
-            app:menu="@menu/drawer_menu" />
+            app:menu="@menu/drawer_menu"
+            app:insetForeground="#4000"/>
```

Comment:
This is transparent trick.
Draw semi-transparent color at the navigation drawer over status bar.
By this, status bar has seen to transparent.

ref:Navigation Drawer semi-transparent over status bar not working
http://stackoverflow.com/questions/26745300/navigation-drawer-semi-transparent-over-status-bar-not-working